charts/seed-controlplane/charts/kube-controller-manager/templates/_helpers.tpl

```diff
@@ -3,3 +3,22 @@
 - --feature-gates={{ range $feature, $enabled := .Values.featureGates }}{{ $feature }}={{ $enabled }},{{ end }}
 {{- end }}
 {{- end -}}
+
+{{- define "kube-controller-manager.controllers" -}}
+{{- if and (semverCompare "< 1.10" .Values.kubernetesVersion) (ne .Values.cloudProvider "") }}
+- --controllers=*,bootstrapsigner,tokencleaner,-service,-route
+{{- else }}
+- --controllers=*,bootstrapsigner,tokencleaner
+{{- end }}
+{{- end -}}
+
+{{- define "kube-controller-manager.cloudProviderFlags" -}}
+{{- if (ne .Values.cloudProvider "") }}
+{{- if semverCompare "< 1.10" .Values.kubernetesVersion }}
+- --cloud-provider={{ .Values.cloudProvider }}
+{{- else }}
+- --cloud-provider=external
```

Comment:
`kube-controller-manager` must have this flag set to `external` as it would start the service/route controller otherwise. Let's not orientate ourselves on this outdated, misleading and wrong documentation file.